tools/src/scie_pants/configure_pants.py

```diff
@@ -122,10 +123,11 @@ def main() -> NoReturn:
         finalizer()
 
     with open(env_file, "a") as fp:
-        print(f"FIND_LINKS={resolve_info.find_links}", file=fp)
+        if resolve_info.find_links:
+            print(f"FIND_LINKS={resolve_info.find_links}", file=fp)
+            print(f"PANTS_SHA_FIND_LINKS={resolve_info.pants_find_links_option(version)}", file=fp)
```

Comment:
Since this binding is used unconditionally as the 1st arg to the `pants` command, it should probably use `--python-repos-find-links=[]` as the else value to avoid passing the empty string as argv1.